### PR TITLE
Linters, HTML and CSS configuration

### PR DESCRIPTION
**Pull request by CesarHerr**
### In this Branch

-  add linters.
-  add a readme file to the project.
- add an index.html to the project.
- add css file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,221 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
```
Comment (Felix45) [inline README.md]: - [ Required ] Please consider removing this instructional text from your readme file, this text serves to guide you on how to fill this section of the readme file. You are supposed to remove the instructional text after filling in this section, please remove it.

File: README.md [hunk 2]
```diff
@@ -1 +1,221 @@
...
+
+If you like this project and you think is useful to someone please share it  
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
```
Comment (Felix45) [inline README.md]: - [ Required ] Please consider removing this instructional text from your readme file, this text serves to guide you on how to fill this section of the readme file. You are supposed to remove the instructional text after filling in this section, please remove it.

File: README.md [hunk 3]
```diff
@@ -1 +1,221 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
Comment (Felix45) [inline README.md]: - [ Required ] Please consider removing this instructional text from your readme file, this text serves to guide you on how to fill this section of the readme file. You are supposed to remove the instructional text after filling in this section, please remove it.

File: README.md [hunk 4]
```diff
@@ -1 +1,221 @@
...
+- Twitter: [@Cesarherr2](https://twitter.com/Cesarherr2)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/cesarherr/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+No features
```
Comment (Felix45) [inline README.md]: - [ Required ] Please include at least two features that you would like to add to this project in the near future. It's good to always show that you envision adding more features to your projects.

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @CesarHerr , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [x] No linters errors :white_check_mark: 
- [x] GitHub flow is used correctly :white_check_mark: 
- [x] Good descriptive commit messages :white_check_mark:   
- [x] Descriptive pull request title and summary :white_check_mark:  

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @CesarHerr, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Removed instructional texts as suggested✔️ 
- Added future features ✔️ 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**